devices/AtomQrCode/AtomQrCode.nuspec

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>nanoFramework.Iot.Device.AtomQrCode</id>
+    <version>$version$</version>
+    <title>nanoFramework.Iot.Device.AtomQrCode</title>
+    <authors>nanoframework</authors>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <license type="file">LICENSE.md</license>
+    <releaseNotes>
+    </releaseNotes>
+    <readme>docs\README.md</readme>
+    <developmentDependency>false</developmentDependency>
+    <projectUrl>https://github.com/nanoframework/nanoFramework.IoT.Device</projectUrl>
+    <icon>images\nf-logo.png</icon>
+    <repository type="git" url="https://github.com/nanoframework/nanoFramework.IoT.Device" commit="$commit$" />
+    <copyright>Copyright (c) .NET Foundation and Contributors</copyright>
+    <description>This package includes the .NET IoT Core binding form M5Stack Atom QR Code for .NET nanoFramework C# projects.</description>
+    <summary>Iot.Device.AtomQrCode assembly for .NET nanoFramework C# projects</summary>
+    <tags>nanoFramework C# csharp netmf netnf m5stack qrcode</tags>
+    <dependencies>
+      <dependency id="nanoFramework.CoreLibrary" version="1.12.0" />
+      <dependency id="nanoFramework.System.IO.Ports" version="1.0.4" />
```

Comment:
Nope. That's pulled by System.IO.Ports

Reply:
hum, ok, I always put the full list there

Reply:
Really no need to. On PR build nanodu will report those that you can remove from the dependency list (because they are a dependency of a 1st level dependency). But no harm done on keeping them there, just it takes longer to update. 😉

Reply:
cool then :-) We have some cleaning work then :-)